src/stores/tests/new-sites-slice.test.ts

```diff
@@ -0,0 +1,65 @@
+import { reducer } from 'src/stores/new-sites-slice';
+
+// Mock ipcListener
+window.ipcListener = {
+	subscribe: jest.fn(),
+};
+
+// Mock getIpcApi
+jest.mock( 'src/lib/get-ipc-api', () => ( {
+	getIpcApi: jest.fn().mockReturnValue( {
+		handleNewSite: jest.fn().mockResolvedValue( undefined ),
+	} ),
+} ) );
+
+describe( 'newSitesSlice', () => {
+	describe( 'reducers', () => {
+		it( 'should add site IDs to processingSiteIds when addProcessingSites is dispatched', () => {
+			const initialState = { processingSiteIds: [] };
+
+			const action = {
+				type: 'newSites/addProcessingSites',
+				payload: [ 'site-1', 'site-2' ],
+			};
```

Comment:
```suggestion
			const action = newSitesActions.addProcessingSites( [ 'site-1', 'site-2' ] );
```
If we export the actions in a single namespace, we can create the action in this way.